### DISCUSSION TITLE
Unable to get predictions after adding customized holidays(events)

### DISCUSSION THREAD

**Discussion by KeplerQzx** [ourownstory/neural_prophet#521] [Q&A - get help using NeuralProphet]
Hi,

I was trying to make a forecast based on previously daily data. Everything looks fine in the fist model.
![image](https://user-images.githubusercontent.com/98067112/150305768-a3ccaedd-edd9-45a4-adc5-6f584bfc6b3d.png)
But when I added two customized holidays to test and rerun the model
![image](https://user-images.githubusercontent.com/98067112/150305967-72b2381c-e9c4-411c-acaa-70618f851b9d.png)
![image](https://user-images.githubusercontent.com/98067112/150306016-bf00c18d-7248-4168-86b5-af6b446439c0.png)
The forecasted data on days after training date becomes NaN
Does anyone have some insight on this? Thanks in advance!

**Comment by poo5297** [comment]
Hi, 
Did you manage to solve this?
If so, could you please share a code snippet?

**Comment by ourownstory** [comment]
@KeplerQzx @poo5297 The specified model here requires 200+ 100 observations *per* sample, however, the dataset seems to be only 99 samples long. Thus, no model can be fitted. It should fail with an error message. If it does not, please share a code snippet that produces this, so we can fix this. Thank you!
